src/redux/csState.ts

```diff
@@ -68,6 +88,13 @@ export function csReducer(state = initialState, action: Action): CsState {
       newValueCache[action.payload.pvName] = newPvState;
```

Comment:
Could this use `updateValueCache` as well?

Reply:
Yep.